pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -449,8 +449,8 @@ message GenerativeSplitAndScatterSpec {
   optional int64 target_size = 15[(gogoproto.nullable) = false];
   // NumEntries is the total number of import spans in this restore.
   optional int64 num_entries = 16[(gogoproto.nullable) = false];
-  // NumNodes is the number of nodes available for dist restore.
-  optional int64 num_nodes = 17[(gogoproto.nullable) = false];
+  // NodeIds is a slice of IDs of nodes available for dist restore.
```

Comment:
to be backward compatible, we cannot do in place replacements of proto fields, so you'll need to add a new field with node ids and leave num nodes alone sadly.

Reply:
Good to know! Will keep this in mind